libs/shared/ui/src/lib/components/callout/callout.tsx

```diff
@@ -0,0 +1,105 @@
+import { type VariantProps, cva } from 'class-variance-authority'
+import { type ComponentPropsWithoutRef, type ElementRef, forwardRef } from 'react'
+import { twMerge } from '@qovery/shared/util-js'
+
+const calloutRootVariants = cva(
+  ['grid', 'items-baseline', 'justify-start', 'gap-x-3', 'p-3', 'border', 'rounded', 'text-sm'],
+  {
+    variants: {
+      color: {
+        green: ['border-green-600', 'bg-green-50', 'text-green-500'],
+        red: ['border-red-500', 'bg-red-50', 'text-red-500'],
+        sky: ['border-sky-500', 'bg-sky-50', 'text-sky-500'],
+        yellow: ['border-yellow-600', 'bg-yellow-50', 'text-yellow-700'],
+        neutral: ['bg-neutral-100', 'border-neutral-300', 'text-neutral-350'],
+      },
+    },
+  }
+)
+
+interface CalloutRootProps
+  extends VariantProps<typeof calloutRootVariants>,
+    Omit<ComponentPropsWithoutRef<'div'>, 'color'> {}
+
+const CalloutRoot = forwardRef<ElementRef<'div'>, CalloutRootProps>(function CalloutRoot(
+  { color, children, className, ...props },
+  ref
+) {
+  return (
+    <div data-accent-color={color} {...props} className={twMerge(calloutRootVariants({ color }), className)} ref={ref}>
+      {children}
+    </div>
+  )
+})
+
+interface CalloutIconProps extends ComponentPropsWithoutRef<'div'> {}
+
+const CalloutIcon = forwardRef<ElementRef<'div'>, CalloutIconProps>(function CalloutIcon(
+  { children, className, ...props },
+  ref
+) {
+  return (
+    <div {...props} className={twMerge('col-start-[-2] text-base', className)} ref={ref}>
```

Comment:
Why you don't manage it with flex and gap? it's not common to have `grid-column-start` at `-2`

Reply:
This is not the same thing. What we need here is to have a consistent display order.
I move to flexbox finally because it was easier on a corner case.
https://github.com/Qovery/console/pull/906/commits/f0abbf345fcdd23d41f0f59aacd98cef0e1e6fc0

Reply:
You've already a consistent display on our DOM, if you remove the `order` properties, it works as well 

Example:
<img width="1559" alt="Capture d’écran 2023-10-26 à 12 52 34" src="https://github.com/Qovery/console/assets/533928/2cc05e14-8ac9-47c6-9bf4-629ae2bac2ac">